Model/Request/Refunds.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace Balancepay\Balancepay\Model\Request;
+
+use Balancepay\Balancepay\Model\AbstractRequest;
+use Balancepay\Balancepay\Model\Response\Factory as ResponseFactory;
+use Magento\Customer\Api\AccountManagementInterface;
+use Magento\Directory\Model\RegionFactory;
+use Magento\Framework\App\RequestInterface;
+use Balancepay\Balancepay\Model\Config;
+use Balancepay\Balancepay\Lib\Http\Client\Curl;
+use Balancepay\Balancepay\Helper\Data as HelperData;
+use Magento\Customer\Model\Session;
+use Magento\Customer\Model\Address;
+use Magento\Framework\Exception\PaymentException;
+
+/**
+ * Balancepay webhooks request model.
+ */
+class Refunds extends AbstractRequest
+{
+    /**
+     * @var RequestInterface
+     */
+    private $request;
+
+    /**
+     * @var Session
+     */
+    private $session;
+
+    /**
+     * @var Address
+     */
+    private $address;
+
+    /**
+     * @var
+     */
+    private $amount;
+
+    private $chargeId;
+
+    /**
+     * Buyers constructor.
+     *
+     * @param Config $balancepayConfig
+     * @param Curl $curl
+     * @param ResponseFactory $responseFactory
+     * @param HelperData $helper
+     * @param AccountManagementInterface $accountManagement
+     * @param RegionFactory $region
+     * @param RequestInterface $request
+     * @param Session $session
+     * @param Address $address
+     */
+    public function __construct(
+        Config $balancepayConfig,
+        Curl $curl,
+        ResponseFactory $responseFactory,
+        HelperData $helper,
+        AccountManagementInterface $accountManagement,
+        RegionFactory $region,
+        RequestInterface $request,
+        Session $session,
+        Address $address
+    ) {
+        $this->request = $request;
+        $this->session = $session;
+        $this->address = $address;
+        parent::__construct(
+            $balancepayConfig,
+            $curl,
+            $helper,
+            $responseFactory,
+            $accountManagement,
+            $region
+        );
+    }
+
+    /**
+     * @var string
+     */
+    protected $_topic;
```

Comment:
why do we have underscore for this property?

Reply:
Removed.